packages/bbui/src/Form/TextArea.svelte

```diff
@@ -5,30 +5,39 @@
 
   export let value: string | undefined = undefined
   export let label: string | undefined = undefined
-  export let labelPosition: string = "above"
+  export let labelPosition = "above"
```

Comment:
Should this be using the position enum, instead?

Reply:
Which enum are you referring to? I dug into the components this gets passed to and they are all either using `string` or not typed yet.

Reply:
Oh... We don't have a type for this. Probably we should add one, but we can leave it outside this PR scope